schema/project-level/project-schema.json

```
@@ -2336,6 +2346,114 @@
         }
       },
       "minProperties": 1
+    },
+    "CostMeasurement": {
+      "title": "Cost measurement",
+      "description": "A measurement of costs for a project.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Local identifier",
+          "description": "A local identifier for this cost measurement.",
```

```suggestion
          "description": "A local identifier for the cost measurement.",
```

A minor style point from OCDS 1.2 style review, amy as well us it here as well